src/site/header.rs

```diff
@@ -17,10 +17,10 @@ async fn fetch_logo(
 ) -> Result<Box<img<String>>> {
     let copy_result = axoasset::copy(&origin_path, dist_dir).await?;
 
-    let path_as_string = copy_result
-        .strip_prefix(dist_dir)
-        .unwrap()
-        .to_string_lossy();
+    let path_as_string = match copy_result.strip_prefix(dist_dir) {
+        Ok(path) => Ok(path.to_string_lossy()),
+        Err(e) => Err(OrandaError::Other(e.to_string())),
```

Comment:
what is this error- could we use a ? instead?

Reply:
changed, added the `StripPrefixError` to our errors

Reply:
danke!